bin/gollum

```diff
@@ -282,7 +282,12 @@ if options[:irb]
   end
 else
   require 'gollum/app'
-  Precious::App.set(:environment, ENV.fetch('RACK_ENV', :production).to_sym)
+  # TODO: Remove RACK_ENV fallback once gollum-6.0-0 is released (4 lines)
+  if Gollum::VERSION < "6.0.0" && ENV['RACK_ENV'] && !ENV['APP_ENV']
+    warn "[DEPRECATION] RACK_ENV will be ignored as of gollum-6.0.0, please use APP_ENV instead."
+    ENV['APP_ENV'] = ENV['RACK_ENV']
```

Comment:
I was surprised to learn that this string comparison works.

If Gollum was a project that had more pre-releases (i.e. `6.0.0.alpha`) I would reach for a more-expressive `Gem::Version` comparison:

```
Gem::Version.new(Gollum::VERSION) < Gem::Version.new("6.0.0")
```

But I manually tested that this does the right thing, and it does. So looks good to me.

Reply:
Actually, I'm not really sure whether checking for the version there is a good idea at all. Might just as well without, provided someone keeps track of such post-release TODOs.

Reply:
I think it could be alright to leave out the version check, if we open an issue to remove the deprecation notice. Even with the check present, we'll have to remember to remove the code af some point, so it doesn't make that much of a difference I think.

So I propose that we keep the deprecation notice (without the version check) until 7.0 and open an issue to remind ourselves to remove the relevant code in that release.

Reply:
@dometto Sorry for the delay, this slipped off my desk somehow. I've removed the version check and just kept the `# TODO` which mentions the lines to be removed once gollum-7 is released.